docs/elasticsearch/snapshot-restore.md

```diff
@@ -216,7 +216,7 @@ storage_class | Specifies the [S3 storage class](https://docs.aws.amazon.com/Ama
 
 ## Take snapshots
 
-You specify two pieces of information when you create a snapshot:
+You specify the following information when you create a snapshot:
```

Comment:
I think this is a regression. That's not to say every list needs a number in front of it, but this list is short and unlikely to change in the future.